azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py

```diff
@@ -3,7 +3,8 @@
 import warnings
 from enum import Enum, IntEnum
 
-from .exceptions import KustoDuplicateMappingError, KustoDuplicateMappingReferenceError, KustoMappingAndMappingReferenceError
+from .exceptions import KustoDuplicateMappingError, KustoDuplicateMappingReferenceError, \
```

Comment:
I think Daniel and Yochai agreed to use line length of 160, this why you have those changes. Sorry I didn't realized this earlier.

Reply:
should i revert ?

Reply:
just run black with --line-length 160